### DISCUSSION TITLE
有办法发送语音或者文件吗？

### DISCUSSION THREAD

**Discussion by Liaozl** [simple-robot/Discussions#31] [Q&A] [answered]
目前找到有发送文字消息和图片的api

**Comment by ForteScarlet** [comment, marked as answer]
文件参考 https://github.com/simple-robot/simpler-robot/discussions/231
语音参考 https://github.com/simple-robot/simbot-component-mirai/blob/main/simbot-component-mirai-core/src/main/kotlin/love/forte/simbot/component/mirai/message/MiraiAudio.kt
